### PR TITLE
Update playwright project

### PR DESCRIPTION
**Pull request by EugenePetrik**
Changes:
- updated playwright dependency
- added @types/node dependency
- updated playwright.config.js
- added vscode/extensions.json
- added GitHub Actions CI config
- updated base fixture
- updated page objects
- added application manager
- updated tests

Tests run - https://github.com/ovartem/playwright_tests_with_po/actions/runs/9927851556
Job run - https://github.com/ovartem/playwright_tests_with_po/actions/runs/9927851556/job/27423339094

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -41,5 +34,6 @@ module.exports = {
             ignoreTemplateLiterals: true,
             ignoreRegExpLiterals: true,
         }],
+        'import/prefer-default-export': 'off' // TODO: remove
```
Comment (ovartem) [inline .eslintrc.js]: I think this should be removed, right?

File: src/fixtures/base.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+// @ts-check
+import { test as base } from '@playwright/test';
+import { Application } from '../pages/Application';
+
+export const test = base.extend({
```
Comment (ovartem) [inline src/fixtures/base.js:5]: can you please setup JSDoc so that intellisense (codecomplition in tests) works for JS code?

**Review by ovartem — CHANGES_REQUESTED** [review]
2 small comments left. Thank you

File: src/fixtures/base.js (hunk 2 above)
Comment (EugenePetrik) [reply src/fixtures/base.js:5]: Thanks for noticing, autocomplete is working now

image

**Review by ovartem — APPROVED** [review]

**Pull request merged**